src/app/main.cpp

```diff
@@ -71,7 +71,8 @@ int main( int argc, char ** argv )
   // Set up the QSettings environment must be done after qapp is created
   QCoreApplication::setOrganizationName( "OPENGIS.ch" );
   QCoreApplication::setOrganizationDomain( "opengis.ch" );
-  QCoreApplication::setApplicationName( "QField" );
+  qDebug() << APP_NAME;
```

Comment:
Leftover?
If it's meaningful to leave, better with some context like `APP_NAME " is starting"`